docker/iota-data-ingestion/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Build image (the specific rust version can also be passed, e.g. "1.82-bookworm")
+ARG RUST_IMAGE_VERSION=bookworm
```

Comment:
why is there no `build.sh` as in all the other docker folders? That takes care to pass the correct environment variables like the git revision or rust toolchain version.

Reply:
[fixed](https://github.com/iotaledger/iota/pull/4946/commits/d3286064cc142e23de6d111fa25111d130aa4e24)